tasks/syncthing.yml

```diff
@@ -0,0 +1,22 @@
+- name: Create Syncthing Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ syncthing_data_directory }}"
+
+- name: Syncthing Docker Container
+  docker_container:
+    name: syncthing
+    image: syncthing/syncthing:latest
+    pull: true
+    network_mode: host
+    volumes:
+      - "{{ syncthing_data_directory }}:/var/syncthing:rw"
+    env:
+      STGUIADDRESS: ""
+    labels:
+      traefik.web.frontend.rule: "Host:syncthing.{{ ansible_nas_domain }}"
```

Comment:
This is correct but you also need to add this hostname to `traefik.toml`, see the file for examples of other applications.